cloud/amazon/sts_session_token.py

```diff
@@ -49,14 +49,14 @@
 '''
 
 RETURN = """
-sts_creds:
+credentials:
     description: The Credentials object returned by the AWS Security Token Service
     returned: always
     type: list
     sample:
-      access_key: ASXXXXXXXXXXXXXXXXXX
+      access_key_id: ASXXXXXXXXXXXXXXXXXX
```

Comment:
This format was chosen for consistency with http://docs.ansible.com/ansible/sts_assume_role_module.html

Will you be changing that module too? I understand the appeal of aligning with the boto3 API, but I'm inclined to think that the train has already left the station when `sts_assume_role` was released.